game2048/src/com/game2048/models/Board.java

```diff
@@ -0,0 +1,26 @@
+package com.game2048.models;
+
+public class Board {
```

Comment:
There is a lot of redundancy in the 2048 code. Try to avoid it as your data can become inconsistent by calling some functions. That should never happen.